excel/validate.py

```diff
@@ -28,17 +16,20 @@ def validate_object(object_validation: dict, object_data: dict):
             if 'mandatory' in attribute_info and not object_has_accession(object_data):
                 mandatory = attribute_info['mandatory'].strip()
                 if mandatory == 'M':
-                    errors.append('Error: Mandatory Atrribute {} is missing.'.format(attribute_name))
+                    errors.append(f'Error: Mandatory Attribute {attribute_name} is missing.')
                 elif mandatory == 'R':
-                    errors.append('Warning: Reccomended Atrribute {} is missing.'.format(attribute_name))
+                    errors.append(f'Warning: Recommended Attribute {attribute_name} is missing.')
                 elif mandatory != 'O':
-                    errors.append('Info: Atrribute {} may be required. {}'.format(attribute_name, mandatory))
+                    errors.append(f'Info: Attribute {attribute_name} may be required. {mandatory}')
```

Comment:
Do we need `{mandatory}` at the end of this message?

Reply:
yep: for example: 'M for paired fastq; NA for single fastq'